packages/nitro-protocol/contracts/ForceMove.sol

```diff
@@ -547,25 +547,25 @@ contract ForceMove is IForceMove {
         return stateHashes;
     }
 
+    enum IsValidTransition {True, NeedToCheckApp}
```

Comment:
nit: `TransitionIsValid` (or something) might be a clearer name for the `True` enum?